<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 22pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>书单列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 75 italic 9pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>双击以预定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="OpenButton">
         <property name="text">
          <string>导入文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_id">
         <property name="styleSheet">
          <string notr="true">font: 22pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>我的书单</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="ChangeButton">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>切换用户</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_2"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 75 italic 9pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>双击以删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CashButton">
         <property name="text">
          <string>结算</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SavaButton">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
